### ISSUE TITLE
Missing Accessibility for changing/selecting payment and shipping

### ISSUE THREAD

**Issue by digitalwert-devs** [shopware/platform#10811]
### Technical TODO

If you select a payment method in the checkout, the page reloads. But what is a really big problem is that the selected payment method is placed right at the top. This has the effect that you can only use the keyboard to jump between the first two entries, as the page reloads immediately when you reach the second entry. 
This urgently needs to be remedied:
No reloading when the payment method is selected (the fields must come with Ajax)
All payment methods always remain in the same order

Version: SW 6.7.0.0

https://github.com/user-attachments/assets/e70ebafe-0d6b-45c8-b41d-7d288cdaf67e

**Comment by mstegmeyer** [comment]
@digitalwert-devs This should be fixed in both 6.6.10 with `ACCESSIBILITY_TWEAKS` feature flag enabled and with 6.7. Are you sure you are testing with `6.7`?

**Comment by mstegmeyer** [comment]
Hey @did you
